docs/content/about/tutorial.md

```diff
@@ -10,63 +10,69 @@ This tutorial covers the basics to get started and to use Nikita. It is organize
 - [What is Nikita?](#what-is-nikita)
 - [Installation instructions](#installation-instructions)
 - [Core concepts](#core-concepts)
-- [Real life example](#real-life-example)
+- [Real-life example](#real-life-example)
 
 Feel free to skip the second section if you are familiar with Node.js and its ecosystem.
 
 For detailed information, navigate the documentation or submit an issue if you don't find the answers to your questions. Also, if you are looking for examples, the source code is well documented and its test coverage quite complete. We highly encourage you to navigate the tests. Tests are self contained and very easy to understand. They also provide you the guaranty of reading a working code.
 
 ## What is Nikita?
 
-Nikita is build as a library, which provides simple functions on a host (server, desktop, machine, vm...) either locally or remotely over ssh.
+Nikita is build as a library, which provides simple functions on a host (server, desktop, machine, vm...) either locally or remotely over SSH.
```

Comment:
Nikita is a toolkit to automate the execution of your workflows. Use one of the many available actions or create your own functions to build simple to complex deployment pipelines and infrastructures. Actions are transparently executed locally or remotely over SSH from any host.